app/Dockerfile

```diff
@@ -10,14 +10,27 @@
 #
 # then visit localhost in a browser (or can use curl from a terminal)
 
-FROM busybox:uclibc AS release
+# Based on the image defined here: https://github.com/AerialTechnologies/docker-alpine-aws-cli/blob/master/Dockerfile
+# and here: https://github.com/AerialTechnologies/docker-alpine-aws-cli-postgresql/blob/master/Dockerfile
```

Comment:
I'm not seeing a connection between these links and the image actually used below `python:alpine`.

Reply:
ah thanks good catch i forgot to change the comment when i changed the app.